### ISSUE TITLE
Missing Create Button

### ISSUE THREAD

**Issue by webdesserts** [beakerbrowser/beaker#782]
Operation System: Mac OS High Sierra
Beaker Version: 0.8 branch

I'm not sure if I should be reporting these since the 0.8 is still a work in progress, but I noticed this one today while using the `DatArchive.selectArchive()` api:

![missing-create-button](https://user-images.githubusercontent.com/916253/33530930-f4ecf93a-d84b-11e7-9e3c-663cf07fb04c.gif)

When you go to select an archive, but decide to "Create new archive", you end up with a modal that has no submit button. I tried enter and that didn't seem to work either.

If you think this is a beginner friendly issue, I could look into this later.

**Comment by taravancil** [comment]
The button [is actually there](https://github.com/beakerbrowser/beaker/blob/beaker-0.8/app/builtin-pages/views/select-archive-modal.js#L134), it's just not visible because the the modal isn't tall enough to show it. If you drag the corner of the modal you'll be able to see the button.

@pfrazee recently made [a fix to master](https://github.com/beakerbrowser/beaker/commit/ac930d2fdbad067fd6f838cf088688413b9a948d) that automatically resizes modals instead of [setting the height manually](https://github.com/beakerbrowser/beaker/blob/beaker-0.8/app/background-process/ui/modals.js#L5). Once master is merged into 0.8 this will no longer be a problem 😄

**Comment by webdesserts** [comment]
Ahh ok. I'll just go back to testing in 0.7 until that merge is in.

**Comment by pfrazee** [comment]
Ok this bugger should be fixed in 0.8 now
